Automattic/jetpack
# AI Assistant: Change tracked event to English instead of translated value and track language and tone change

Fixes #32284
Fixes #32285

## Proposed changes:

* Changes the value used in the `jetpack_editor_ai_assistant_block_toolbar_button_click` event to the original string in English.
* Tracks tone and language change

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Make sure the `debug` messages are visible with `localStorage.setItem( 'debug', '*' );` and refreshing the page
* Add an AI Assistant block
* Ask for a translation or tone change
* Check that there is an event log for each action
* Click on `Write with AI...`
* Click on one of the suggestions in the `Write…` section, like `Post about...`
* Check that there is an event log in English regardless of the language of the blog

![2023-08-07_14-09-08](https://github.com/Automattic/jetpack/assets/8486249/0bfb4bab-17c3-43d4-918d-2445f7ad2059)

![2023-08-04_16-11-44](https://github.com/Automattic/jetpack/assets/8486249/4fcdab0f-a64b-44d2-a46d-5e4c71667fc9)
![2023-08-04_16-11-15](https://github.com/Automattic/jetpack/assets/8486249/9b8f431e-2940-409a-b221-67885d43f5d0)
![2023-08-04_16-10-20](https://github.com/Automattic/jetpack/assets/8486249/a8b07895-cf85-401d-b2e7-0953a5564f23)
